tests/css-pseudo-4.js

```diff
@@ -29,6 +29,13 @@ export default {
 			},
 			tests: ['::selection'],
 		},
+		'::search-text': {
+			links: {
+				tr: '#selectordef-search-text',
```

Comment:
Until those new features made it into a TR publication, only the draft links should be provided.

Reply:
Thanks, done